### PR TITLE
Fix: return 404 for not existing assets

### PR DESCRIPTION
**Pull request by PhilippKilian**

### CODE REVIEW

File: b3lb/rest/views.py [hunk 1]
```diff
@@ -118,24 +117,39 @@ def metrics(request, slug=None, sub_id=0):
...
 # Endpoint for getting logos
 @require_http_methods(['GET'])
 def logo(request, slug=None):
     try:
-        return utils.get_file_response_from_storage(Asset.objects.get(tenant__slug=slug.upper()).logo.name)
+        asset = Asset.objects.get(tenant__slug=slug.upper())
     except ObjectDoesNotExist:
         return HttpResponseNotFound()
 
+    if asset.custom_css:
```
Comment (liske) [inline b3lb/rest/views.py:138]: Should it be `logo`?

File: b3lb/rest/views.py [hunk 2]
```diff
@@ -118,24 +117,39 @@ def metrics(request, slug=None, sub_id=0):
 @require_http_methods(['GET'])
 def slide(request, slug=None):
     try:
-        return utils.get_file_response_from_storage(Asset.objects.get(tenant__slug=slug.upper()).slide.name)
+        asset = Asset.objects.get(tenant__slug=slug.upper())
     except ObjectDoesNotExist:
         return HttpResponseNotFound()
 
+    if asset.custom_css:
```
Comment (liske) [inline b3lb/rest/views.py:124]: Should it be `slide`?

**Review by liske — CHANGES_REQUESTED** [review]

File: b3lb/rest/views.py (hunk 1 above)
Comment (PhilippKilian) [reply b3lb/rest/views.py]: yes... fixed

File: b3lb/rest/views.py (hunk 2 above)
Comment (PhilippKilian) [reply b3lb/rest/views.py]: yes, fixed

**Review by liske — APPROVED** [review]

**Pull request merged**